### ISSUE TITLE
Forge Extension by GPT

### ISSUE THREAD

**Issue by rauldlnx10** [captainzero93/extract-unet-safetensor#2]
If you want to add it as Forge Extension... 

[sd-webui-forge-extract-unet](https://github.com/rauldlnx10/sd-webui-forge-extract-unet)

**Comment by lokitsar** [comment]
Worked like a charm and I was also able to see what I was doing wrong originally when I wasn't getting the application to work.  Thank you for this!

**Comment by rauldlnx10** [comment]
> Worked like a charm and I was also able to see what I was doing wrong originally when I wasn't getting the application to work. Thank you for this!

Based in the latest version.
[unet-extractor.zip](https://github.com/user-attachments/files/16650454/unet-extractor.zip)

**Comment by rauldlnx10** [comment]
> Worked like a charm and I was also able to see what I was doing wrong originally when I wasn't getting the application to work. Thank you for this!

[unet-extractor.zip](https://github.com/user-attachments/files/16651031/unet-extractor.zip)

**Comment by wbclark** [comment]
If you want to release it as a Forge extension, the proper way to do so is to release the source code on GitHub so that people can know what they are getting before they download and extract a .zip file with completely unknown and potentially insecure contents.

> Worked like a charm and I was also able to see what I was doing wrong originally when I wasn't getting the application to work. Thank you for this!

The quoted user has very little if any visible activity on this platform, and _could_ quite possibly be a sock-puppet account operated for the purpose of making potential attack vectors like the above .zip file seem seem legit, which is a social engineering tactic that was recently made popular by the famous attack that inserted malicious code into the XZ library.

The fact that interest in diffusion models and other local generative AI tooling has recently attracted a lot of new people to this platform, unfortunately means that it is also attractive for those looking to prey on people who are not following security best practices.

I **strongly advise** anyone reading this _against_ downloading and extracting these files; instead, wait for a proper release of the extension code from the author, which if they have good intentions they will surely provide in order to avoid the possible perception of doing something nefarious.

Please use your brains, remain vigilant, and think twice before downloading anything of unknown origin and contents, most of all a zip file purporting to be an otherwise unreleased extension for a project that is very popular right now with people who are very new to this technology.

**Comment by captainzero93** [comment]
> If you want to release it as a Forge extension, the proper way to do so is to release the source code on GitHub so that people can know what they are getting before they download and extract a .zip file with completely unknown and potentially insecure contents.
> 
> > Worked like a charm and I was also able to see what I was doing wrong originally when I wasn't getting the application to work. Thank you for this!
> 
> The quoted user has very little if any visible activity on this platform, and _could_ quite possibly be a sock-puppet account operated for the purpose of making potential attack vectors like the above .zip file seem seem legit, which is a social engineering tactic that was recently made popular by the famous attack that inserted malicious code into the XZ library.
> 
> The fact that interest in diffusion models and other local generative AI tooling has recently attracted a lot of new people to this platform, unfortunately means that it is also attractive for those looking to prey on people who are not following security best practices.
> 
> I **strongly advise** anyone reading this _against_ downloading and extracting these files; instead, wait for a proper release of the extension code from the author, which if they have good intentions they will surely provide in order to avoid the possible perception of doing something nefarious.
> 
> Please use your brains, remain vigilant, and think twice before downloading anything of unknown origin and contents, most of all a zip file purporting to be an otherwise unreleased extension for a project that is very popular right now with people who are very new to this technology.

this is correct , please create a repo [rauldlnx10](https://github.com/rauldlnx10)

my licence just asks for attribution , double check it yourself before uploading your rebuilt extension

**Comment by rauldlnx10** [comment]
> If you want to release it as a Forge extension, the proper way to do so is to release the source code on GitHub so that people can know what they are getting before they download and extract a .zip file with completely unknown and potentially insecure contents.
> 
> > Worked like a charm and I was also able to see what I was doing wrong originally when I wasn't getting the application to work. Thank you for this!
> 
> The quoted user has very little if any visible activity on this platform, and _could_ quite possibly be a sock-puppet account operated for the purpose of making potential attack vectors like the above .zip file seem seem legit, which is a social engineering tactic that was recently made popular by the famous attack that inserted malicious code into the XZ library.
> 
> The fact that interest in diffusion models and other local generative AI tooling has recently attracted a lot of new people to this platform, unfortunately means that it is also attractive for those looking to prey on people who are not following security best practices.
> 
> I **strongly advise** anyone reading this _against_ downloading and extracting these files; instead, wait for a proper release of the extension code from the author, which if they have good intentions they will surely provide in order to avoid the possible perception of doing something nefarious.
> 
> Please use your brains, remain vigilant, and think twice before downloading anything of unknown origin and contents, most of all a zip file purporting to be an otherwise unreleased extension for a project that is very popular right now with people who are very new to this technology.

I apologize if I caused any concern by uploading the .zip file, I understand some users concerns due to possible attacks. I created the extension with very little programming knowledge and with the help of ChatGPT. I do this out of curiosity and as a hobby. I use github for my little projects.

As captainzero93 suggested I have created a repository. Thanks for everything :)

[sd-webui-forge-extract-unet](https://github.com/rauldlnx10/sd-webui-forge-extract-unet)

**Comment by rauldlnx10** [comment]
> > If you want to release it as a Forge extension, the proper way to do so is to release the source code on GitHub so that people can know what they are getting before they download and extract a .zip file with completely unknown and potentially insecure contents.
> > > Worked like a charm and I was also able to see what I was doing wrong originally when I wasn't getting the application to work. Thank you for this!
> > 
> > 
> > The quoted user has very little if any visible activity on this platform, and _could_ quite possibly be a sock-puppet account operated for the purpose of making potential attack vectors like the above .zip file seem seem legit, which is a social engineering tactic that was recently made popular by the famous attack that inserted malicious code into the XZ library.
> > The fact that interest in diffusion models and other local generative AI tooling has recently attracted a lot of new people to this platform, unfortunately means that it is also attractive for those looking to prey on people who are not following security best practices.
> > I **strongly advise** anyone reading this _against_ downloading and extracting these files; instead, wait for a proper release of the extension code from the author, which if they have good intentions they will surely provide in order to avoid the possible perception of doing something nefarious.
> > Please use your brains, remain vigilant, and think twice before downloading anything of unknown origin and contents, most of all a zip file purporting to be an otherwise unreleased extension for a project that is very popular right now with people who are very new to this technology.
> 
> this is correct , please create a repo [rauldlnx10](https://github.com/rauldlnx10)
> 
> my licence just asks for attribution , double check it yourself before uploading your rebuilt extension

Done, :) thanks for the advice

**Comment by wbclark** [comment]
Well done, that's a much better way to distribute it.

Thanks for listening and taking the concern seriously.
